src/mlagility/api/api.py

```diff
@@ -170,7 +171,9 @@ def benchit(
         gpu_model = gpumodel.load(
             gmodel.state.config.build_name, cache_dir=gmodel.state.cache_dir
         )
-        perf = gpu_model.benchmark()
+
+        perf = gpu_model.benchmark(backend=backend)
```

Comment:
This is ok for this PR, but before release we need to support arbitrary IP addresses (see #6)